Runtime/Scripts/GltfImport.cs

```diff
@@ -685,7 +685,14 @@ public UnityEngine.Material GetMaterial(int index = 0)
         {
             if (m_Materials != null && index >= 0 && index < m_Materials.Length)
             {
```

Comment:
Some kind of switch or feature flag may be useful, if we still want to use the glTF converted materials (PBR only)?

Reply:
maybe like we do it here https://github.com/area28technologies/io.area28.unity/blob/6c7e5e9a8baf4a1a9ec65fa71bb772e503f95a35/src/package/Runtime/LivePlayManager.cs#L8 👍

Reply:
All right, I'll use a scripting define symbol for this.

Reply:
Done. I used `UNITY_MATERIALS_ENABLED`.